src/utils/MessageBuilder.ts

```diff
@@ -97,6 +100,59 @@ export default class MessageBuilder {
     };
   }
 
+  static reportedTeamButtons(buttonInteraction: ButtonInteraction, activeMatchEmbed: MessageEmbed): MessageOptions {
+    const primaryStyle = "PRIMARY";
+    const reportBlue = new MessageButton({
+      customId: ButtonCustomID.ReportBlue,
+      label: "🔷 Blue Team Won 🔷",
+      style: "SECONDARY",
+    });
+    const reportOrange = new MessageButton({
+      customId: ButtonCustomID.ReportOrange,
+      label: "🔶 Orange Team Won 🔶",
+      style: "SECONDARY",
+    });
+
+    let reportedTeam;
+    switch (buttonInteraction.customId) {
+      case ButtonCustomID.ReportBlue: {
+        reportBlue.style = primaryStyle;
```

Comment:
Is there a `.setStyle()` method we could use instead of mutating the property directly?

Reply:
Resolved.